app/components/works/keywords_row_component.html.erb

```diff
@@ -1,14 +1,15 @@
 <div class="mb-3 row plain-container g-0 keyword-row">
   <div class="col-sm-6">
-    <%= form.label :label, "Keyword", class: "col-form-label visually-hidden" %>
-    <div data-controller="keywords autocomplete autocomplete-edit" data-autocomplete-url-value="/autocomplete" data-autocomplete-min-length-value="3" class="dropdown" role="combobox">
-      <%= form.text_field :label, {class: "form-control#{"is-invalid" if error?}", required: true, "data-autocomplete-target": "input", "data-autocomplete-edit-target": "input",
+    <% unique_field_id = "work_keywords_attributes_#{form.options[:child_index]}" %>
+    <div data-controller="keywords autocomplete autocomplete-edit" data-autocomplete-url-value="/autocomplete" data-autocomplete-min-length-value="3" aria-label="autocomplete group" class="dropdown" role="combobox" aria-expanded="false">
```

Comment:
it seemed weird that there were complaints about a div not having a label, but ... doing this got rid of the error (siteImprove, as I recall)